nncf/helpers/requirements.txt

```diff
@@ -0,0 +1,3 @@
+-c ../../constraints.txt
```

Comment:
I would suggest to remove requirements.txt and implement lazy checking of installed packages. Many repositories use this approach for additional requirements, such as `transformers`, `optimum` and etc.

Reply:
Could you please provide the proper example? As I understand, the current approach is not valid for you.

Reply:
Example: 
https://github.com/huggingface/optimum/blob/2fb5ea5ca7ca8ea887af2851cce80ab2545d3f4f/optimum/gptq/quantizer.py#L742

Reply:
Done.